cmd/incusd/main_forknet.go

```diff
@@ -135,7 +135,11 @@ static void forkdonetdhcp() {
 	}
 
 	// Set the process title.
-	(void)setproctitle("[incus DHCP] eth0");
+	char *workdir = advance_arg(false);
+	char *title = malloc(sizeof(char)*strlen(workdir)+18);
+	sprintf(title, "[incus dhcp] %s eth0", workdir);
```

Comment:
You're not accounting for the \0 in the length of title.

Reply:
Doh, I thought I did but apparently I don't know how to count, will fix...